backend/app/api/endpoints/projects.py

```
@@ -140,6 +142,59 @@ def archive_all_project_chats_endpoint(
+    db: Session = Depends(get_db),
+):
+    """List Wework worktree directories by scanning each relevant online device once."""
+    try:
+        return await project_service.list_project_worktrees(
+            db=db,
+            user_id=current_user.id,
+            client_origin=client_origin,
+        )
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)